### ISSUE TITLE
WARNING   [cache] Failed to save key "xxx"

### ISSUE THREAD

**Issue by wenmingtang** [symfony/symfony#29118]
**Symfony version(s) affected**: 4.2.0-beta1

PHP Version: 7.2

**Description**  
```
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CComment%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CComment%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CUser%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CUser%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CArticle%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CArticle%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CTag%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CTag%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CCategory%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CCategory%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CTopic%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CTopic%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
17:56:47 WARNING   [cache] Failed to save key "%5BApp%5CEntity%5CGroup%24CLASSMETADATA%5D%5B1%5D" (Doctrine\ORM\Mapping\ClassMetadata) ["key" => "%5BApp%5CEntity%5CGroup%24CLASSMETADATA%5D%5B1%5D","type" => "Doctrine\ORM\Mapping\ClassMetadata","exception" => Symfony\Component\Cache\Exception\InvalidArgumentException { …}]
```

**How to reproduce**  

```
php bin/console  doctrine:schema:validate
```

<img width="1439" alt="screen shot 2018-11-07 at 5 58 50 pm" src="https://user-images.githubusercontent.com/1620172/48124192-abdb4780-e2b6-11e8-9d6c-10038bec55ea.png">

**Comment by nicolas-grekas** [comment]
Can you provide a reproducer please? Can you maybe debug that InvalidArgumentException?
The easiest way to do so might be to use dump() in the CacheItem::log() method.

**Comment by wenmingtang** [comment]
@nicolas-grekas 
Thank you for your help.
See the output below:
```php

    /**
     * Internal logging helper.
     *
     * @internal
     */
    public static function log(LoggerInterface $logger = null, $message, $context = array())
    {
dump($context); // debug
        if ($logger) {
            $logger->warning($message, $context);
        } else {
            $replace = array();
            foreach ($context as $k => $v) {
                if (is_scalar($v)) {
                    $replace['{'.$k.'}'] = $v;
                }
```

<img width="1084" alt="wx20181112-092942 2x" src="https://user-images.githubusercontent.com/1620172/48321469-a3d82a80-e65d-11e8-8d1d-b10dfe24cfb5.png">

**Comment by nicolas-grekas** [comment]
Fixed in #29209
Thanks for the report and the insights!
